src/types/big-serial.ts

```diff
@@ -0,0 +1,26 @@
+import * as TypeUtils from '../type-utils';
+
+const mysqlType: TypeUtils.DatabaseTypeFn = (
+	necessity: string,
+	index: string,
+	defaultValue = '',
+) => 'BIGINT' + defaultValue + necessity + index + ' AUTO_INCREMENT';
+mysqlType.castType = 'BIGINT';
+
+const websqlType: TypeUtils.DatabaseTypeFn = (
+	necessity: string,
+	index: string,
+	defaultValue = '',
+) => 'BIGINT' + defaultValue + necessity + index + ' AUTOINCREMENT';
+websqlType.castType = 'BIGINT';
+
+export const types = {
+	postgres: 'BIGSERIAL',
+	mysql: mysqlType,
+	websql: websqlType,
+	odata: {
+		name: 'Edm.Int64',
+	},
+};
+
+export const validate = TypeUtils.validate.integer;
```

Comment:
We should probably consider switching this type & BigInt to use the new `BigInt` js type, since `parseInt()` fails if a number > 32 bits is passed.